java/core/src/test/java/co/worklytics/psoxy/storage/StorageHandlerTest.java

```diff
@@ -98,4 +99,10 @@ void buildRequest(String inputBasePath, String outputBasePath, String expectedOu
         assertEquals("bucket", request.getDestinationBucketName());
         assertEquals(expectedOutputPath, request.getDestinationObjectPath());
     }
+
+    @Test
+    public void metadata() {
+        handler.getObjectMeta("bucket", "directory/file.csv", handler.buildDefaultTransform());
```

Comment:
what's the test? missing assertions

Reply:
yeah, added something. but kinda lame.

what I really want to test is that metadata works round trip - eg, we can set it, then get it back through AWS as expected so `hasBeenSanitized` call is correct.